test/Table_Tests/src/Common_Table_Operations/Conversion_Spec.enso

```diff
@@ -547,6 +549,16 @@ spec setup =
             t.auto_value_types . should_fail_with Unsupported_Database_Operation
             t.at "X" . auto_value_type . should_fail_with Unsupported_Database_Operation
 
+    if setup.is_database then Test.group prefix+"Table.from_objects" <|
+        Test.specify "should report unsupported" <|
+            Database_Table.from_objects 12 . should_fail_with Unsupported_Database_Operation
```

Comment:
I guess this test case needs to be removed too

Reply:
Done